client/src/components/Main.jsx

```diff
@@ -303,13 +320,30 @@ function Main() {
             <Typography id="modal-modal-description" sx={{ mt: 2 }}>
               <div>
                 <Avatar alt={profile.name} src={profile.image} />
```

Comment:
Would it be possible to add a placeholder profile image to fallback on if profile.image isn't defined?